nf_core/lint.py

```diff
@@ -451,6 +486,25 @@ def check_nextflow_config(self):
             else:
                 self.passed.append((4, "Config variable process.container looks correct: '{}'".format(container_name)))
 
+    def check_actions_branch_protection(self):
+        """Checks that the github actions workflows are valid.
+
+        Makes sure that ``nf-core lint``runs in github actions workflows and that 
+        tests run with the required nextflow version.
+        """
+        for wf in ['.github/workflows/branch.yml']:
+            fn = os.path.join(self.path, wf)
+            if os.path.isfile(fn):
+                with open(fn, 'r') as fh:
+                    branchwf = yaml.safe_load(fh)
+                # Check that we have the master branch protection, but allow patch as well
+                branchMasterCheck = 'master'
+                try:
+                    assert(branchMasterCheck in branchwf.get('on', {'pull_request', {'branches', {}}}))
```

Comment:
Instead of using `get` here I think we can just try to access the key and handle the `KeyError` if it doesn't work..

```suggestion
                    assert(branchMasterCheck in branchwf['on']['pull_request']['branches'])
```